### ISSUE TITLE
NewGameMenu + StartMenu

### ISSUE THREAD

**Issue by LordStephen77** [DreamBlocks/DreamBlocks#44]
The game menu is divided into two very similar classes: NewGameMenu and StartMenu. The main game menu is divided into 2 different phases:

- Home screen
- Second screen after pressing the "new" button.

I would like to unify the NewGameMenu and StartMenu class to simplify the customization of the game menu.

Or I can post comments to the code to remember all the steps.

New collaborators reading reviews will have easy life.

Comments are important! We comment on the code as I did at the beginning.

I probably realized a new package devoted to these two classes. I want to keep everything in order, and separate the menu from the rest of the project. I'm working on the new buttons, and the new header.

If you have good ideas, tips and more, I would be happy to share your thoughts. Thank you.

**Comment by AKryukov92** [comment]
What exactly do you want to customize and unify?
What is common in those two menu?
Why do you want additional package? They already in "com.lordstephen77.dreamblocks.ui".

**Comment by LordStephen77** [comment]
@AKryukov92 I solved! I just have to put comments on the code to make it more understandable. Now I work on the main menu, I want to put the new buttons I've drawn, and apply the resume button when I press ESC.

**Comment by AKryukov92** [comment]
I don't think that comments like "Size inventory panel and block cells" add much to code.
First of all, field "tileSize" do not define actual size of whole panel.
Its size is defined by variables "panelHeight" and "panelWidth", which names are very obvious.
Comment "Size inventory panel and block cells" is misleading.

Comment in this line `private int tileSize = 25; // Cells size` don't add much either. If existing variable name is not clear for you, then rename variable. Though, I don't think that "cellSize" is much better than "tileSize".

Fragments like
```
/**
     * 
     * @param g
     * @param screenWidth
     * @param screenHeight
     */
```
produce inspection warning

> '@param g' tag description is missing

If you add them, then write description too
```
    /**
     * 
     * @param g - instance of GraphicsHandler
     * @param screenWidth - current width of screen
     * @param screenHeight - current height of screen
     */
```

**Comment by LordStephen77** [comment]
I put those comments for my convenience, I want to comment on the rest of the code, I needed to remember everything. My memory is not very good. I also found references to pointer and crafting, I have to work calmly, I put everything flat.

Do not mind these things, I only take small notes to help remind me of what has been done in the past.
